test/tests/neutronics/feedback/lattice/openmc.i

```diff
@@ -62,16 +62,7 @@
   [heat_source]
     type = ElementIntegralVariablePostprocessor
     variable = heat_source
-  []
```

Comment:
couldn't a fix to avoid a re-gold be to have the tallies exist on blocks 1, 2, and 3? Sometimes our tests become slightly more verbose/not recommended approach in the interest of avoiding regolds.

Reply:
And I would say in this case if we _can_ avoid a re-gold of exodus, to do so.